src/main/ccl/cclutassert_impl_nff.inc

```diff
@@ -97,27 +97,51 @@ subroutine cclut::evaluateComparison(cclutLineNum, cclutCtx, cclutComparison, cc
   set cclutReturnValue = FALSE
   set cclutCondition = build2('"', cclutVal1, '" ', cclutComparison, ' "', cclutVal2, '"')
   if (cclutComparison = "=")
-    if (cclutVal1 = cclutVal2)
+    if (
+         cclutVal1 = cclutVal2
+         and
+         (textlen(cclutVal1) = textlen(cclutVal2) or cclut::ignoreCclStringLogic)
```

Comment:
"ignore CCL string logic" is not the best name for this.
When it is TRUE, CCL string logic \***is\*** used.